ToolBox/IvolatilityEquityConverter/Program.cs

```diff
@@ -0,0 +1,165 @@
+            Console.WriteLine("==============================================");
+            Console.WriteLine(
+                "The IVolatility converter transforms IVolatility orders into the LEAN Algorithmic Trading Engine Data Format.");
+            Console.WriteLine("Two parameters are required: ");
+            Console.WriteLine("   1> Source archived IVolatility Data.");
+            Console.WriteLine("   2> Destination Directory of LEAN Data Folder. (Typically located under Lean/Data)");
+            Console.WriteLine("   3> Resolution of your IVolatility data. (min,hour,day)");
+            Console.WriteLine(" ");
+            Console.WriteLine("NOTE: THIS WILL OVERWRITE ANY EXISTING FILES.");
+
+            if (args.Length == 3)
+            {
+                _sourceDirectory = args[0];
+                _destinationDirectory = args[1];
+                _resolution = ParseResolution(args[2]);
+            }
+            else
+            {
+                Console.WriteLine("1. Ivolatility equity data source directory: ");
+                _sourceDirectory = (Console.ReadLine() ?? "");
+                Console.WriteLine("2. Destination LEAN Data directory: ");
+                _destinationDirectory = (Console.ReadLine() ?? "");
+                Console.WriteLine("3. Enter resolution of source data (min, hour, day): ");
+                _resolution = ParseResolution(Console.ReadLine() ?? "");
+            }
+
+            //Count the total files to process:
+            Console.WriteLine("Counting Files..." + _sourceDirectory);
+            var count = 1;
+            var files = GetFiles(_sourceDirectory);
+            Console.WriteLine("Processing {0} Files ...", files.Length);
+
+            foreach (var file in files)
+            {
+                Console.WriteLine("Processing {0} of {1} files ...", count, files.Length);
+                var symbol = GetSymbol(file);
+                var streamProvider = StreamProvider.ForExtension(Path.GetExtension(file));
+                var inputStream = streamProvider.Open(file).First();
+                var streamReader = new StreamReader(inputStream);
+                var datawriter = new LeanDataWriter(_resolution, symbol, _destinationDirectory);
+                IList<TradeBar> fileBars = new List<TradeBar>();
+                string line;
+                while ((line = streamReader.ReadLine()) != null)
+                {
+                    var linearray = ParseCsv(line);
+                    if (linearray.Length <= 2) continue;
+                    var time = DateTime.ParseExact(linearray[TimeField], DateFormat.UI,
+                        CultureInfo.InvariantCulture);
+                    var priceBid = Decimal.Parse(linearray[PriceBidField]);
+                    var sizeBid = Decimal.Parse(linearray[SizeBidField]);
+                    var priceAsk = Decimal.Parse(linearray[PriceAskField]);
+                    var sizeAsk = Decimal.Parse(linearray[SizeAskField]);
```

Comment:
The quote bar information will be lost in the trade bar file. Maybe there's enough information to write QuoteBars for equity as well?

Reply:
We can write both `QuoteBar` and `TradeBar`, but for quotes all 4 OHLC values will be identical as mentioned in the ticket. I will add support for both.

Reply:
@mchandschuh There's a problem writing Quote bars due to invalid cast when writing equity quotes for minute resolution. Issue is, it tries to cast to `TradeBar` even though im providing it quote bars. So for this current PR I will ignore and comment on it in the code.

```
case SecurityType.Equity:
                    switch (resolution)
                    {
                        case Resolution.Tick:
                            var tick = (Tick) data;
                            return ToCsv(milliseconds, Scale(tick.LastPrice), tick.Quantity, tick.Exchange, tick.SaleCondition, tick.Suspicious ? "1" : "0");

                        case Resolution.Minute:
                        case Resolution.Second:
                            var bar = (TradeBar) data;
                            return ToCsv(milliseconds, Scale(bar.Open), Scale(bar.High), Scale(bar.Low), Scale(bar.Close), bar.Volume);

                        case Resolution.Hour:
                        case Resolution.Daily:
                            var bigBar = (TradeBar) data;
                            return ToCsv(longTime, Scale(bigBar.Open), Scale(bigBar.High), Scale(bigBar.Low), Scale(bigBar.Close), bigBar.Volume);
                    }
                    break;
```